lib/api/src/store.rs

```diff
@@ -23,7 +22,7 @@ pub struct Store {
     engine: Arc<dyn Engine + Send + Sync>,
     tunables: Arc<dyn Tunables + Send + Sync>,
     #[loupe(skip)]
```

Comment:
It may be possible to remove `#[loupe(skip)]` here I guess (if `TrapHandler` can be inspected).

Reply:
Edit: Nah, it's not possible because `TrapHandler` is a type alias to `dyn Fn`. Forget about it!